tests/ops/op_math/test_adjoint_op.py

```diff
@@ -579,8 +578,7 @@ def test_queueing_base_defined_outside(self):
         with qml.tape.QuantumTape() as tape:
             op = Adjoint(base)
 
-        assert len(tape) == 1
-        assert tape.get_info(op)["owns"] is base
+        assert qml.queuing.AnnotatedQueue.__len__(tape) == 1
```

Comment:
At this point we should just change the test to convert to created Queues and Scripts. I can make this official in #3454 or in a follow-up PR, so I'm really just here to say I regret introducing this pattern :p

Reply:
Basically, if we're testing queuing behaviour, use `AnnotatedQueue`.  I'll fix that up.